### PR TITLE
Adding documentation to `README.md`

### PR DESCRIPTION
**Pull request by pgoiporia**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,13 @@
-This package is used to access SuperstaQ via a Web API through Qiskit
\ No newline at end of file
+This package is used to access SuperstaQ via a Web API through Qiskit
+
+Please note that Python version `3.8` or higher is required. qiskit-superstaq and all of its
```
Comment (vtomole) [inline README.md:3]:
```suggestion
Please note that Python version `3.7` or higher is required. qiskit-superstaq and all of its
```

File: README.md [hunk 2]
```diff
@@ -1 +1,13 @@
...
+dependencies can be installed via:
+
+```
+git clone https://github.com/SupertechLabs/qiskit-superstaq.git
```
Comment (vtomole) [inline README.md:7]: We don't need to git clone in the README. Can just to `pip install cirq-superstaq`. Try to make this file as similar to https://github.com/quantumlib/Cirq/blob/master/README.rst as feasible..

**Review by vtomole — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -1 +1,40 @@
...
+
+```
+./dev_tools/build_docs
+open docs/build/html/index.html # For Mac
+xdg-open docs/build/html/index.html # For Linux
+```
+
+### Unit Tests
+
+In order to get unit tests to work locally, you must:
```
Comment (singular-value) [inline README.md:33]: I don't think this is true 🤔

Comment (pgoiporia) [reply README.md:33]: ahaha yeah still in draft version, was just copying over docs from SuperstaQ for a template

File: README.md [hunk 4]
```diff
@@ -1 +1,42 @@
...
+pip install -r dev-requirements.txt
+pip install -e .
+```
+
+Make sure to always be in the qiskit-superstaq-env (via ``conda activate qiskit-superstaq-env``) when you're working on qiskit-superstaq.
+
+### Creating and submitting a circuit through qiskit-superstaq
+```
+import qiskit
+import qiskit_superstaq as qss
```
Comment (vtomole) [inline README.md:23]: Remove the `as qss`

File: README.md [hunk 5]
```diff
@@ -1 +1,42 @@
...
+This package is used to access SuperstaQ via a Web API through Qiskit. Qiskit programmers
+can take advantage of the applications, pulse level optimization, and write-once-target-all
+features of SuperstaQ.
+
+
+Please note that Python version `3.7` or higher is required. qiskit-superstaq and all of its
+dependencies can be installed via:
+
+```
+git clone https://github.com/SupertechLabs/qiskit-superstaq.git
```
Comment (vtomole) [inline README.md:10]: Replace with `pip install qiskit-superstaq` and also replace conda stuff with `venv`

File: README.md [hunk 6]
```diff
@@ -1 +1,42 @@
-This package is used to access SuperstaQ via a Web API through Qiskit
\ No newline at end of file
+This package is used to access SuperstaQ via a Web API through Qiskit. Qiskit programmers
+can take advantage of the applications, pulse level optimization, and write-once-target-all
+features of SuperstaQ.
```
Comment (vtomole) [inline README.md:3]:
```suggestion
features of SuperstaQ with this package.
```

**Review by vtomole — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (vtomole) [reply README.md:10]:
```suggestion
```

File: README.md [hunk 7]
```diff
@@ -1 +1,42 @@
...
+can take advantage of the applications, pulse level optimization, and write-once-target-all
+features of SuperstaQ with this package.
+
+
+Please note that Python version `3.7` or higher is required. qiskit-superstaq and all of its
+dependencies can be installed via:
+
+```
+git clone https://github.com/SupertechLabs/qiskit-superstaq.git
+cd qiskit-superstaq
```
Comment (vtomole) [inline README.md:11]:
```suggestion
```

File: README.md [hunk 8]
```diff
@@ -1 +1,42 @@
...
+python3 -m venv venv_qiskit_superstaq
+source venv_qiskit_superstaq/bin/activate
+pip install -r dev-requirements.txt
+pip install -e .
```
Comment (vtomole) [inline README.md:15]:
```suggestion
```

File: README.md [hunk 9]
```diff
@@ -1 +1,42 @@
...
+
+Please note that Python version `3.7` or higher is required. qiskit-superstaq and all of its
+dependencies can be installed via:
+
+```
+git clone https://github.com/SupertechLabs/qiskit-superstaq.git
+cd qiskit-superstaq
+python3 -m venv venv_qiskit_superstaq
+source venv_qiskit_superstaq/bin/activate
+pip install -r dev-requirements.txt
```
Comment (vtomole) [inline README.md:14]:
```suggestion
pip install qiskit-superstaq
```

**Review by vtomole — CHANGES_REQUESTED** [review]

File: README.md [hunk 10]
```diff
@@ -1 +1,41 @@
...
+This package is used to access SuperstaQ via a Web API through Qiskit. Qiskit programmers
+can take advantage of the applications, pulse level optimization, and write-once-target-all
+features of SuperstaQ with this package.
+
+
+Please note that Python version `3.7` or higher is required. qiskit-superstaq and all of its
+dependencies can be installed via:
+
+```
+pip install qiskit-superstaq
```
Comment (vtomole) [inline README.md:10]: Move this command to after the env has been activated.

File: README.md [hunk 11]
```diff
@@ -1 +1,41 @@
...
+python3 -m venv venv_qiskit_superstaq
+source venv_qiskit_superstaq/bin/activate
+```
+
+Make sure to always be in the qiskit-superstaq-env (via ``source venv_qiskit_superstaq/bin/activate``) when you're working on qiskit-superstaq.
```
Comment (vtomole) [inline README.md:15]:
```suggestion
```

**Review by vtomole — CHANGES_REQUESTED** [review]

**Review by vtomole — DISMISSED** [review]
LGTM

**Review by vtomole — APPROVED** [review]

**Pull request merged**